inst/schema/repository_fetch_response.json

```diff
@@ -0,0 +1,4 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object"
```

Comment:
Eventually we may want to add stuff like how long it took to do the fetch in the response, but not something that is needed yet.